src/gateways/MsSQL/retrieveActiveDepartmentsByOrganisationId.js

```diff
@@ -4,8 +4,9 @@ export default ({ getMsSqlConnPool }) => async (organisationId) => {
   const res = await db
     .request()
     .input("organisation_id", mssql.Int, organisationId)
+    .input("status", mssql.TinyInt, 1)
```

Comment:
You'll want to use the `statusToId()` function here instead of `1`

Reply:
done.